src/util/bip158.rs

```diff
@@ -190,13 +199,13 @@ impl <'a> BlockFilterWriter<'a> {
 
 
 /// Reads and interpret a block filter
-pub struct BlockFilterReader {
```

Comment:
Oh, why did you make these ones private? I thouht the Reader was so that a user could use the match method without having to re-allocate the bytes. Like this they still have to. Leave these public and the user has the choice to just allocate a `BlockFilter` or to wrap their existing bytes with the Reader.

Reply:
ok

Reply:
wait. there is no savings there. nothing can be reused

Reply:
made public again